jscs/auto-fix.js

```diff
@@ -0,0 +1,14 @@
+// Add support for a --noFix argument to disable JSCS auto-fix.
+
+module.exports = jscsAutoFix;
+
+function jscsAutoFix() {
+  var args = process.argv.slice(2);
+  var noFixArgIndex = args.indexOf('--noFix');
+  var noFix = noFixArgIndex > -1;
+
+  if (noFix) {
+    process.argv.splice(noFixArgIndex + 2, 1);
```

Comment:
we should handle automatically stripping arguments that minimist expects and pass that to `fashion-show` rather than modifying `process.argv`. That can be handled before or after the minimist bit.